### PR TITLE
rhol crc/manage secrets

### PR DESCRIPTION
**Pull request by cjeanner**
This PR has two distinct patches in it:

- Removed useless files in manage_secrets
A tiny change that probably doesn't needs its own pull-request,
just a small cleaning

- Switch rhol_crc to the new manage_secrets
The important part - switching to the new manage_secrets role
for the sake of consistency and stability.

As a pull request owner and reviewers, I checked that:
- [X] Appropriate testing is done and actually running
- [X] Appropriate documentation exists and/or is up-to-date:
  - [X] README in the role

### CODE REVIEW

**Comment by cjeanner** [comment]
recheck

**Comment by cjeanner** [comment]
recheck

**Comment by cjeanner** [comment]
recheck

File: ci_framework/roles/rhol_crc/README.md [hunk 1]
```diff
@@ -22,6 +22,16 @@ Become - required for the tasks in `sudoers_grant.yml` and `sudoers_revoke.yml`
...
 * `cifmw_rhol_crc_creds`: (Boolean) Add crc creds to bashrc. Defaults to `False`.
+* `cifmw_rhol_crc_pullsecret_dest`: (String) Absolute path of the pull-secrets destination. Defaults to `~/pull-secret.json`.
+
+### Pull-secret management
+This role calls the [manage_secrets](./manage_secrets.md) role. It allows to copy or create
+the pull-secret on the target host, in a known location. In order to run the `rhol_crc`
+role, you **must** provide either `cifmw_manage_secrets_pullsecret_file` OR
+`cifmw_manage_secrets_pullsecret_content`.
+
+If you provide neither, it will fail. If you provide both, it will also fail.
```
Comment (marios) [inline ci_framework/roles/rhol_crc/README.md:33]: If you provide neither, or both, it will fail.

**Review by marios — APPROVED** [review]
looks right/didn't spot something

File: ci_framework/roles/rhol_crc/README.md (hunk 1 above)
Comment (pablintino) [reply ci_framework/roles/rhol_crc/README.md:33]: Agree on this one

**Comment by pablintino** [comment]
/approve

Update the small English nit on a PR if you want

**Pull request merged**